### ISSUE TITLE
Windows ARM64 support / using device's gyroscope for motion games

### ISSUE THREAD

**Issue by holaimscott** [mgba-emu/mgba#2494]
I wanted to see if there was a way to make mGBA run natively on ARM64. While GBA emulation is not intensive, this feature would still be cool so that ARM processing is not as intense on the system.

Additionally, I wanted to ask if mGBA already has a feature to use a win tablet's gyroscope to be used in games that require motion control?

**Comment by endrift** [comment]
The biggest thing preventing me from shipping ARM64 support is that there isn't really a good way to build non-MSVC projects for ARM64 Windows yet. I think gcc doesn't target it yet or something. Once that's resolved I'd absolutely love to add it, but that's been holding me back.

Try using a dev build to see if the tablet's gyros work. They may. The dev builds use SDL's sensor integration, so I'd have to see how SDL handles that.

**Comment by Testsr** [comment]
gcc can build for ARM64 [targets](https://archlinux.org/packages/community/x86_64/aarch64-linux-gnu-gcc/), but MSYS 2 has not added that target yet.

**Comment by endrift** [comment]
Yes, ARM64 targets, but not ARM64 Windows. Just because you can spit out ARM64 binaries doesn't mean it'll necessarily adhere to the ABI used in a particular OS, or have support for the OS's interfaces, which is the problem here.

**Comment by Pokemetti** [comment]
I'm pretty sure Clang already can target Windows on ARM, is there anything wrong with compiling with Clang for Windows on ARM until GCC adds support? Technically there might also be some unofficial GCC builds that can target Window on ARM, but that feels rather hacky.
